plugins/parodos/src/models/project.ts

```diff
@@ -8,17 +8,26 @@ const projectStatus = z.union([
   z.literal('COMPLETED'),
 ]);
 
+const accessRole = z.union([
+  z.literal('Owner'),
+  z.literal('Developer'),
+  z.literal('Admin'),
+]);
+
 export type ProjectStatus = z.infer<typeof projectStatus>;
 
 export const projectSchema = z.object({
   id: z.string(),
   name: z.string(),
   description: z.string().nullable(),
-  createDate: z.coerce.date(),
-  modifyDate: z.coerce.date(),
+  createdDate: z.coerce.date(),
+  modifiedDate: z.coerce.date(),
   status: projectStatus
     .default('IN_PROGRESS')
     .transform(value => value.split('_').join(' ')),
+  modifiedBy: z.string().nullable(),
+  createdBy: z.string().nullable(),
+  accessRole: accessRole.transform(value => value.split('_').join(' ')),
```

Comment:
```suggestion
  accessRole,
```
I don't think we need the transform as none of the elements of the `accessRole` string union  have a `_` in them.

Reply:
Ack. I just thought to e future proof and standard with other enums.

Reply:
that is a valid point.  Feel free to leave it in if you want